app/screens/latex/latex.js

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    BackHandler,
+    ScrollView,
+    Text,
+    View,
+    Platform,
+} from 'react-native';
+import MathView from 'react-native-math-view';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import {popTopScreen} from '@actions/navigation';
+import {splitLatexCodeInLines} from '@utils/latex';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+export default class Latex extends React.PureComponent {
+    static propTypes = {
+        theme: PropTypes.object.isRequired,
+        content: PropTypes.string.isRequired,
+    };
+
+    componentDidMount() {
+        BackHandler.addEventListener('hardwareBackPress', this.handleAndroidBack);
+    }
+
+    componentWillUnmount() {
+        BackHandler.removeEventListener('hardwareBackPress', this.handleAndroidBack);
+    }
+
+    handleAndroidBack = () => {
+        popTopScreen();
+        return true;
+    };
+
+    render() {
+        const style = getStyleSheet(this.props.theme);
+
+        const lines = splitLatexCodeInLines(this.props.content);
+
+        return (
+            <SafeAreaView
+                edges={['bottom', 'left', 'right']}
```

Comment:
why is top is being omitted?

Reply:
I don't know why the top is omitted but i based my code on the `Code` screen and there it is omitted too.